bleachbit/Windows.py

```diff
@@ -202,11 +200,11 @@ def delete_registry_key(parent_key, really_delete):
 
 def delete_updates():
     """Returns commands for deleting Windows Updates files"""
-    windir = expandvars('$windir')
+    windir = bleachbit.expandvars('$windir')
```

Comment:
We can't use `from bleachbit import expandvars` because that'd cause a circular import